src/Service/ServiceFactory.php

```diff
@@ -172,6 +172,19 @@ public function getProgrammesService(): ProgrammesService
         return $this->instances['ProgrammesService'];
     }
 
+    public function getPromotionsService(): PromotionsService
+    {
+        if (!isset($this->instances['PromotionsService'])) {
```

Comment:
I love constants for these cases, but I decided to follow they way is done so far. In the factory appears simple strings to define service names